hawtio-system/src/main/java/io/hawt/web/FileUploadFilter.java

```diff
@@ -0,0 +1,5 @@
+package io.hawt.web;
+
+public interface FileUploadFilter {
```

Comment:
Why is it needed?

Reply:
This is just to introduce a `type` for the implementation instance, in this case `GlobalFileUploadFilter`, this is to ensure if people wants to introduce different filter/behavior then they stick to `FileUploadFilter` category.

Reply:
IMO so long as the interface doesn't have any methods, it's not useful. If your intention is to provide users an extension point for behaviours then such a marker interface (= an interface without behaviours) are somewhat odd. If you just want to make a category for filters then just digging a `uploadfilter` package is enough.

Reply:
OK will remove it.